### PR TITLE
BP-872 - Add Scheduler project to the Blueprint Template

### PR DESCRIPTION
**Pull request by laslorajda**
BP-872

### CODE REVIEW

File: Enigmatry.Entry.Blueprint.Scheduler/Program.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+var builder = Host.CreateDefaultBuilder(args);
+
+builder.UseSerilog((context, services, configuration) =>
+    configuration
+        .ReadFrom.Configuration(context.Configuration)
+        .ReadFrom.Services(services));
+
+builder.ConfigureServices((context, services) =>
+{
+    using var factory = new SerilogLoggerFactory();
```
Comment (zsrdjan) [inline Enigmatry.Entry.Blueprint.Scheduler/Program.cs:15]: Is this scheduler configured completely. I.e. can you resolve IRepository, send MediatR request?

File: Enigmatry.Entry.Blueprint.Scheduler/Jobs/EntryBlueprintExampleJob.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private readonly ILogger<EntryBlueprintExampleJob> _logger;
+
+    public EntryBlueprintExampleJob(ILogger<EntryBlueprintExampleJob> logger, IConfiguration configuration) : base(logger, configuration)
+    {
+        _logger = logger;
+    }
+
+    public override Task Execute(EmptyJobRequest request)
+    {
+        _logger.LogInformation("Scheduled job executed at {Now}", DateTimeOffset.Now);
```
Comment (zsrdjan) [inline Enigmatry.Entry.Blueprint.Scheduler/Jobs/EntryBlueprintExampleJob.cs:19]: Can we create a job that will create a user in the local database?

File: Enigmatry.Entry.Blueprint.Scheduler/Program.cs (hunk 1 above)
Comment (zsrdjan) [reply Enigmatry.Entry.Blueprint.Scheduler/Program.cs:15]: Scheduler.Tests project is missing.

**Review by zsrdjan — CHANGES_REQUESTED** [review]

File: Enigmatry.Entry.Blueprint.Scheduler.Tests/SchedulerFixtureBase.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    .Where(t => t.Name.EndsWith("Job", StringComparison.InvariantCulture)).AsSelf();
+
+                container.RegisterModule(new EntityFrameworkModule { RegisterMigrationsAssembly = true });
+            });
+
+    private void SeedTestUser()
+    {
+        var dbContext = Resolve<AppDbContext>();
+        var testUser = TestUserData.CreateSchedulerUser();
+        if (dbContext.Set<User>().SingleOrDefault(x => x.Id == testUser.Id) != null)
```
Comment (laslorajda) [inline Enigmatry.Entry.Blueprint.Scheduler.Tests/SchedulerFixtureBase.cs:63]: I don't love this, but seeding/migration will create this user initially and then respawn will delete it. Couldn't think of a better solution.

**Review by zsrdjan — APPROVED** [review]

**Pull request merged**